atlas/templates/Jenkinsfile.gotmpl

```diff
@@ -54,7 +54,7 @@ pipeline {
             dir ("${WORKSPACE}/${DIRECTORY}") {
                 withDockerRegistry([credentialsId: "<insert-the-creds-id>", url: ""]) {
                     withAWS(region:'<insert-the-region-id>', credentials:'<insert-the-creds-id>') {
-                        sh "IMAGE_VERSION=\$(IMAGE_VERSION)-j$BUILD_NUMBER make push-chart"
```

Comment:
Yeah, this seems like a bug. `IMAGE_VERSION` needs to be consistent throughout `build` and `push-chart` stages. Good eye. 👍